docs/developers_guide/micro_frontends_in_open_edx.rst

```diff
@@ -16,35 +16,168 @@ To run an MFE locally you must install `node 12 <https://nodejs.org>`_ (with npm
   npm run test
   npm run build (build static output)
 
+****************************************
 Configuration with Environment Variables
-----------------------------------------
+****************************************
 
-Open edX MFEs expect a number of process environment variables (Accessible in node via ``process.env``) in order to properly configure authentication, routing to other parts of the system, etc. Each MFE has ``.env``, ``.env.development``, and ``.env.test`` files that are consumed by the MFE. The production .env file contains only null values – the build process should supply them via the command line prior to running ``npm run build``.
+Open edX MFEs expect a number of process environment variables (Accessible in node via ``process.env``) in order to properly configure the application. This includes links to other parts of the system, branding elements like the site name and logos, cookie names, and authentication configuration details. Each MFE has ``.env``, ``.env.development``, and ``.env.test`` files that are consumed by the MFE. The production ``.env`` file contains only falsy values – the build process should supply them via the command line prefixing the ``npm run build`` command.
 
-Example ``.env.development``:
+Production build example::
 
-.. code-block:: bash
+  SITE_NAME="My Site Name" npm run build
+
+The above would only set the SITE_NAME - suitable values for the environment variables should be supplied as well.
```

Comment:
for `all` the env variables? for `other` env variables? (this is a language nit)